app/lib/history/models.dart

```diff
@@ -290,18 +258,6 @@ class UploaderInvited implements HistoryEvent {
   }
 
   Map<String, dynamic> toJson() => _$UploaderInvitedToJson(this);
-
-  @override
-  HistoryEvent migrateUser(
```

Comment:
Let's retain this constructor and document it deprecated, leave a comment that we're only keeping for documenting the existing datastore..